packages/label-sync/src/app.ts

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Application } from 'probot';
+import { request } from 'gaxios';
+import { GCFBootstrapper } from 'gcf-utils';
+import { GitHubAPI } from 'probot/lib/github';
+import {labels} from './labels.json';
+
+const reposUrl = 'https://github.com/googleapis/sloth/blob/master/repos.json';
+
+interface RepoResponse {
+  repos: {
+    language: string;
+    repo: string;
+  }[]
+}
+
+const appFn = async (app: Application) => {
+
+  // Sync labels when a new repo is created
+  app.on('repository.created', async context => {
+    const owner = context.payload.repository.owner.login;
+    const repo = context.payload.repository.name;
+    await reconcileLabels(owner, repo, context.github);
+  });
+
+  // On startup, synchronize labels for all relevant repos
+  const reposResponse = await request<RepoResponse>({
+    url: reposUrl
+  });
+  await Promise.all(reposResponse.data.repos.map(r => {
+    const [owner, repo] = r.repo.split('/');
+    return reconcileLabels(owner, repo, {} as any);
```

Comment:
When this bot wakes up for the first time, I'd like to iterate over all repos and run this little reconciler thingy.  After that, we'll just look at net new repos.  When someone modifies the `labels.json`, my assumption is that this would get re-deployed, and restart the server, running this thingy again.  Here's the only problem.  I don't seem to have access to the `context.github` object with the preloaded `octokit` instance like I have on event hooks.  

Do y'all have any ideas on the "right" way to do this?
@orthros @bcoe